src/library_html5.js

```diff
@@ -1670,14 +1671,14 @@ var LibraryJSEvents = {
     // Queue this function call if we're not currently in an event handler and the user saw it appropriate to do so.
     if (!canPerformRequests) {
       if (strategy.deferUntilInEventHandler) {
-        JSEvents.deferCall(JSEvents.requestFullscreen, 1 /* priority over pointer lock */, [target, strategy]);
+        JSEvents.deferCall(_JSEvents_requestFullscreen, 1 /* priority over pointer lock */, [target, strategy]);
         return {{{ cDefine('EMSCRIPTEN_RESULT_DEFERRED') }}};
       } else {
         return {{{ cDefine('EMSCRIPTEN_RESULT_FAILED_NOT_DEFERRED') }}};
       }
     }
 
-    return JSEvents.requestFullscreen(target, strategy);
+    return _JSEvents_requestFullscreen(target, strategy);
```

Comment:
why not make this depend on `$JSEvents`? is it to save code size I guess?

how about doing `$JSEvents_requestFullscreen` for readability, like other helper functions?

Reply:
The intent is for this symbol to be private, so do not want to `$` that one. These needed to be pulled away from JSEvents so that JS linker would be able to track these without pulling the whole JSEvents object in.